mindsdb/integrations/handlers/athena_handler/athena_handler.py

```diff
@@ -62,7 +62,7 @@ def connect(self) -> StatusResponse:
             return StatusResponse(self.name, True, 'Connected successfully')
```

Comment:
Can we please update this too? Also there are 2 more places we need to update in this class

Reply:
done !!